requirements/common.txt

```diff
@@ -1,31 +1,31 @@
-beautifulsoup4==4.4.0
-django-admin-sortable2==0.6.4
+beautifulsoup4==4.4.1
+django-admin-sortable2==0.6.7
 django-allauth==0.23.0
 django-angular==0.8.4
 django-appconf==1.0.1
 django-classy-tags==0.7.2
-django-cms==3.3.3
+https://github.com/jrief/django-cms/archive/f9b62031f64aa7d775f9d1befa7b6a70d0d1a719.zip#egg=django-cms
 django-compressor==1.6
 django-filer==1.2.5
 django-filter==0.15.3
 django-fsm==2.4.0
 django-fsm-admin==1.2.3
 django-haystack==2.5.0
 django-ipware==1.1.1
-django-mptt==0.8.6
+django-mptt==0.8.7
 django-parler==1.6.4
 django-polymorphic==1.0.2
 django-post-office==2.0.8
 django-rest-auth==0.5.0
 django-sass-processor==0.5.3
 django-sekizai==0.10.0
 Django-Select2==5.8.9
-djangoshop-stripe==0.2.0
+https://github.com/awesto/djangoshop-stripe/archive/257a33f17df7f6d42e6d9815969ec6c801dcc66c.zip#egg=djangoshop-stripe
```

Comment:
sure, but then I would publish a release which does not work with the current version of django-SHOP.